### PR TITLE
Relative Timestamp fixes

### PR DESCRIPTION
**Pull request by lbwexler**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [X] Caught up with `develop` branch as of last change.
- [X] Added CHANGELOG entry, or determined not required.
- [X] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [X] Updated doc comments / prop-types, or determined not required.
- [X] Reviewed and tested on Mobile, or determined not required.
- [X] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: cmp/relativetimestamp/RelativeTimestamp.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ function doFormat(opts) {
     const {prefix, equalString, epsilon, allowFuture, short} = opts,
         diff = opts.relativeTo - opts.timestamp,
         elapsed = Math.abs(diff),
-        isEqual =  elapsed < epsilon * SECONDS,
+        isEqual = elapsed <= (epsilon ?? 0) * SECONDS,
```
Comment (haynesjm42) [inline cmp/relativetimestamp/RelativeTimestamp.js]: Why the `?? 0` here? We default it to 10 above, do we expect someone to pass `null`?

**Comment by lbwexler** [comment]
Particularly concerned about the 'undefined'
yes, easy to do when you have the code (like our client app) that sometimes
passes one, sometimes, doesn't.

**Comment by haynesjm42** [comment]
But we default it to `10` in `getRelativeTimestamp`, so if no `epsilon` is passed then we are covered there, not sure this is a case we really need to worry about? It's not going to hurt anything of course, but should we at least be defaulting to `10` instead of `0` to be consistent?

**Comment by lbwexler** [comment]
I want someone who tried to turn 'epsilon off' with an explicit setting to null or undefined to get zero.  That is wha they are expecting

**Pull request merged**